packages/pdk-nag/scripts/generate-pack-markdown.ts

```diff
@@ -0,0 +1,29 @@
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0 */
+import { NagMessageLevel } from "cdk-nag";
+import * as fs from "fs-extra";
+import * as Mustache from "mustache";
+import { PackName, RuleMetadata } from "../src/packs/aws-prototyping-rules";
+
+const renderedMarkdown = Mustache.render(
+  fs
+    .readFileSync(
+      `${__dirname}/templates/markdown/awsprototypingrules-readme.mustache`
```

Comment:
Can you please confirm the mustache file is present in `dist/js/pdk-nag@0.0.0.jsii.tgz`

Reply:
Within `pdk-nag@0.0.0.jsii.tgz` the `awsprototypingrules-readme.mustache` file is present in `/packages/pdk-nag/dist/js/package/scripts/templates/markdown`